files/image_config/platform/rc.local

```diff
@@ -114,6 +114,9 @@ migrate_nos_configuration()
         # remove nos-config-part from cmdline
         sed -r -i.bak "s/nos-config-part=[^[:space:]]+//" /host/grub/grub.cfg
 
+        # remove ssd-upgrader-part from cmdline
```

Comment:
could you please add a comment here to show what exactly the string replacement is?

Reply:
In this command, the `ssd-upgrader-part` option is completely removed from the command line (replaced with an empty string) similar to nos-config-part.
Please let me know if the comment has to updated with 'empty string' in place of 'remove'.

Reply:
I see, current comment matches the  ode, then I am fine with it.